tests/test_plots.py

```diff
@@ -1,238 +1,318 @@
+  of the plot function (plot behavior) that changed.
+"""
+# global parameters -----------------------------------------------------------
+create_baseline = False
+
+# file type used for saving the plots
+file_type = "png"
+
+# if true, the plots will be compared to the baseline and an error is raised
+# if there are any differences. In any case, differences are written to
+# output_path as images
+compare_output = False
+
+# path handling
+base_path = os.path.join('tests', 'test_plot_data')
+baseline_path = os.path.join(base_path, 'baseline')
+output_path = os.path.join(base_path, 'output')
+
+if not os.path.isdir(base_path):
+    os.mkdir(base_path)
+if not os.path.isdir(baseline_path):
+    os.mkdir(baseline_path)
+if not os.path.isdir(output_path):
+    os.mkdir(output_path)
+
+# remove old output files
+for file in os.listdir(output_path):
+    os.remove(os.path.join(output_path, file))
+
+# the naming scheme if the baseline is as follows:
+# <test_name>_<function_name>_<parameters>.png
+
+# testing ---------------------------------------------------------------------
+@pytest.mark.parametrize('function', [
+    (sp.plot.scatter),
+    (sp.plot.voronoi_cells_sphere)])
+def test_sampling_scatter(function):
+    """Test the scatter plot with default arguments."""
+    coords = sp.samplings.equal_area(n_max=0, n_points=12)
+
+    # do plotting
+    filename = f'{function.__name__}_default'
+    create_figure()
+    function(coords)
+    save_and_compare(
+        create_baseline, baseline_path, output_path, filename,
+        file_type, compare_output)
+
+    match = 'must be a coordinates object.'
+    with pytest.raises(ValueError, match=match):
+        function('coords')
+
+
+@pytest.mark.parametrize('function', [
+    (sp.plot.balloon),
+    (sp.plot.balloon_wireframe),
+    (sp.plot.contour),
+    (sp.plot.contour_map),
+    (sp.plot.pcolor_map),
+    (sp.plot.pcolor_sphere)])
+def test_spherical_default(function):
+    """Test all spherical plots with default arguments."""
+    coords = sp.samplings.equal_area(n_max=0, n_points=500)
+    data = np.sin(coords.colatitude) * np.cos(coords.azimuth)
+
+    # do plotting
+    filename = f'default_{function.__name__}'
+    create_figure()
+    function(coords, data)
+    save_and_compare(
+        create_baseline, baseline_path, output_path, filename,
+        file_type, compare_output)
+
+    # test error for invalid inputs
+    match = 'data must be a 1D array with the same cshape as the coordinates'
+    with pytest.raises(ValueError, match=match):
+        function(coords, 'data')
+
+    match = 'coordinates must be a coordinates object.'
+    with pytest.raises(ValueError, match=match):
+        function('coords', data)
+
+
+@pytest.mark.parametrize('function', [
+    (sp.plot.balloon),
+    (sp.plot.balloon_wireframe),
+    (sp.plot.contour),
+    (sp.plot.contour_map),
+    (sp.plot.pcolor_map),
+    (sp.plot.pcolor_sphere)])
```

Comment:
```suggestion
    (sp.plot.contour),
    (sp.plot.contour_map),
    (sp.plot.pcolor_map)])
```
Since the colormap handling depends on the phase (or in the future cmap_encoding) I would not test these here, but move to a different test.
I'm already implementing such a test as part of https://github.com/pyfar/spharpy/pull/184